src/applications/disability-benefits/2346/containers/ConfirmationPage.jsx

```diff
@@ -15,36 +19,95 @@ export class ConfirmationPage extends React.Component {
   componentDidMount() {
     focusElement('.schemaform-title > h1');
     scrollToTop();
+    this.props.getVeteranInformationData();
   }
 
   render() {
+    const date = new Date();
+    const { email } = this.props;
+
     return (
       <>
-        <h2>
-          Your order is confirmed
-          <i className="form2346 fas fa-check-circle fa-4x" />
-        </h2>
+        <AlertBox
+          headline="Your order is confirmed"
+          content={
+            <p>
+              We'll send you an email confirming your order to{' '}
+              <strong>{email}</strong>.
+            </p>
+          }
+          status="success"
+        />
+        <AlertBox
+          headline="Your order details"
+          content={
+            <section>
+              <p>
+                <strong>Order Date</strong>:{' '}
+                {new Intl.DateTimeFormat('en-US', {
+                  month: 'long',
+                  day: 'numeric',
+                  year: 'numeric',
+                }).format(date)}
+              </p>
```

Comment:
I'm not sure if we should be using `Intl.DateTimeFormat` yet. It is supported in most browsers, including IE11, but I don't think it's standard practice. Usually `moment` is used for this type of formatting.